words/english-borrowed.txt

```diff
@@ -761,6 +761,7 @@
 ലാർവ
 ലാറ്ററൈറ്റ്
 ലാറ്റിറ്റ്യൂഡ്
+ലിവ് 
```

Comment:
ലിവ്  or ലീവ് ?  
I haven't seen ലിവ് being used in Malayalam. but ലീവ് is commonly used as part of Malayalam. 
eg: നാളെ ലീവാണോ?